### DISCUSSION TITLE
How to get project roles using v2 API?

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8812] [Q&A] [answered]
Hello! I need help finding a v2 API method for [searching project roles](https://zitadel.com/docs/apis/resources/mgmt/management-service-list-project-roles). Could you please point me to the documentation? Thank you.

**Comment by hifabienne** [comment, marked as answer]
Hi @ashep 
That request is not yet migrated to the v2 api.
You will have to use the request you have sent

**Reply by ashep** [reply]
Thank you. Is there any ETA for this?
